### PR TITLE
Scrub custom-words.txt for non-words and descriptions for more inclusive terminology

### PR DESCRIPTION
**Pull request by TimLovellSmith**
## ARM API Information (Control Plane)

<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow.
</i>
Azure 1st Party Service can try out the [Shift Left](https://aka.ms/ShiftLeft) experience to initiate API design review from ADO code repo.  If you are interested, may request engineering support by filling in with the form https://aka.ms/ShiftLeftSupportForm.

### Changelog
Add a changelog entry for this PR by answering the following questions:
  1. What's the purpose of the update?
      - [ ] new service onboarding
      - [ ] new API version
      - [ ] update existing version for new feature
      - [ ] update existing version to fix swagger quality issue in s360
      - [x] Other, please clarify --> updating documentation, capitalization, naming, and/or spell-checking rules
  2. When are you targeting to deploy the new service/feature to public regions? Please provide the date or, if the date is not yet available, the month. (N/A)
  3. When do you expect to publish the swagger? Please provide date or, the date is not yet available, the month. (N/A)
  4. By default, Azure SDKs of all languages (.NET/Python/Java/JavaScript for both management-plane SDK and data-plane SDK, Go for management-plane SDK only ) MUST be refreshed with/after swagger of new version is published. If you prefer NOT to refresh any specific SDK language upon swagger updates in the current PR, please leave details with justification here. (N/A - can be picked up in future SDK versions)

### Contribution checklist (MS Employees Only):
- [x] I commit to follow the [Breaking Change Policy](https://aka.ms/AzBreakingChangesPolicy) of "no breaking changes"
- [x] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [ ] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist

> **Applicability**: :warning:
>
> If your changes encompass only the following scenarios, you should SKIP this section, as these scenarios do not require ARM review.
> - Change to data plane APIs
> - Adding new properties
> - All removals

Otherwise your PR may be subject to ARM review requirements. Complete the following:
- [ ] Check this box if any of the following apply to the PR so that the label "ARMReview" and "WaitForARMFeedback" will be added by bot to kick off ARM API Review.  Missing to check this box in the following scenario may result in delays to the ARM manifest review and deployment.
  - Adding a new service
  - Adding new API(s)
  - Adding a new API version
    -[ ] To review changes efficiently, ensure you copy the existing version into the new directory structure for first commit and then push new changes, including version updates, in separate commits. You can use OpenAPIHub to initialize the PR for adding a new version. For more details refer to the [wiki](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/208/OpenAPI-Hub-Adding-new-API-version).

- [ ] Ensure you've reviewed following [guidelines](https://aka.ms/rpguidelines) including [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc) and [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). Estimated time (4 hours). This is required before you can request review from ARM API Review board.

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them.

### Breaking Change Review Checklist
If you have any breaking changes as defined in the [Breaking Change Policy](https://aka.ms/AzBreakingChangesPolicy/), request approval from the Breaking Change Review Board.

**Action**: to initiate an evaluation of the breaking change, create a new intake using the [template for breaking changes](https://aka.ms/Breakingchangetemplate). Additional details on the process and office hours are on the [Breaking Change Wiki](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37684/Breaking-Changes).

NOTE: To update API(s) in public preview for over 1 year (refer to [Retirement of Previews](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37683/Retirement-of-Previews))

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

**Comment by TimLovellSmith** [comment]
@tadelesh What's the right way to validate this PR - does the swagger spellcheck actually check spellings of files that aren't even touched in the PR? Is there way to force real spellchecking of everything?

**Comment by TimLovellSmith** [comment]
@tadelesh I guess one pragmatic point of view is, it doesn't even matter if I removed too many words, because anyone who has spellcheck fail on their future PR where they touched an existing .json, can easily just add back any terms that they need. They might be unpleasantly surprised that they had to do it, of course.

**Comment by TimLovellSmith** [comment]
@raych1  What's the right way to validate this PR - does the swagger spellcheck actually check spellings of files that aren't even touched in the PR? Is there way to force real spellchecking of everything?

Or can we just merge it without further validations?

AFAICS, it is pretty low impact if I made a mistake and removed an extra word, because anyone who has spellcheck fail on their PR can just add it straight back.

**Comment by ruowan** [comment]
This PR LGTM.  The spell check in PR only validate the PR changed files.  For check everything, maybe we could build an offline pipeline to check.

File: specification/vmwarecloudsimple/resource-manager/Microsoft.VMwareCloudSimple/stable/2019-04-01/vmwarecloudsimple.json [hunk 1]
```diff
@@ -3016,7 +3016,7 @@
           }
         },
         "vrOpsEnabled": {
-          "description": "Is Vrops enabled/disabled",
+          "description": "Is vRealize Operations Manager enabled/disabled",
```
Comment (raych1) [inline specification/vmwarecloudsimple/resource-manager/Microsoft.VMwareCloudSimple/stable/2019-04-01/vmwarecloudsimple.json:3019]: How do you get this name `vRealize`?

File: cSpell.json [hunk 2]
```diff
@@ -818,7 +825,7 @@
             ]
         },
         {
-            "filename": "**/specification/web/resource-manager/Microsoft.Web/stable/2020-12-01/CommonDefinitions.json",
+            "filename": "**/specification/web/resource-manager/Microsoft.Web/*/*/CommonDefinitions.json",
```
Comment (raych1) [inline cSpell.json:828]: Can you use ** to replace */*?

File: custom-words.txt [hunk 3]
```diff
@@ -4,7 +4,6 @@ A256CBCPAD
 AADDS
 aadiam
 AADKERB
```
Comment (raych1) [inline custom-words.txt:6]: How do you know these words to be deleted are not used in swaggers anymore?

File: specification/vmwarecloudsimple/resource-manager/Microsoft.VMwareCloudSimple/stable/2019-04-01/vmwarecloudsimple.json (hunk 1 above)
Comment (TimLovellSmith) [reply specification/vmwarecloudsimple/resource-manager/Microsoft.VMwareCloudSimple/stable/2019-04-01/vmwarecloudsimple.json:3019]: I googled 'vrops' and guessed it refers to this. Though apparently its the old product name, now. https://www.vmware.com/products/vrealize-operations.html.

File: cSpell.json (hunk 2 above)
Comment (TimLovellSmith) [reply cSpell.json:828]: Sure, I don't see a problem with that.

File: custom-words.txt (hunk 3 above)
Comment (TimLovellSmith) [reply custom-words.txt:6]: Also by manual review, I spotted a few 'obvious typos' like 'Seleted'.

**Review by raych1 — APPROVED** [review]

**Comment by TimLovellSmith** [comment]
@raych1 I've resolved conflicts the easy way - keeping any words that looked like they might be added. If you are OK with merging this, could you please? I don't have that access.

**Comment by TimLovellSmith** [comment]
Resolved merged conflicts. Still look good? @tadelesh

**Comment by TimLovellSmith** [comment]
@lirenhe resolved conflicts again. Is it good to merge?

**Review by lirenhe — APPROVED** [review]

**Comment by lirenhe** [comment]
@akning-ms, I talked to @raych1 and this PR is mainly for the wording change in description so it is good to merge.
I am unable to bypass SDK generation failure in Go/python track2,  could you help to force merge this PR?  thanks

**Comment by TimLovellSmith** [comment]
@akning-ms resolved conflicts again, is this good to go?

**Comment by TimLovellSmith** [comment]
Thanks @akning-ms !

**Pull request merged**